.github/workflows/validate-pull-request.yml

```diff
@@ -31,9 +31,10 @@ jobs:
       with:
         mode: exactly
         count: 1
-        labels: breaking, feature, BIGBUG, bug
+        labels: breaking, feature, BIGBUG, bug, dependencies
 
     - name: Pull requests must link an issue
+      if: ${{ contains(github.event.issue.labels.*.name, 'breaking') || contains(github.event.issue.labels.*.name, 'feature') || contains(github.event.issue.labels.*.name, 'BIGBUG') || contains(github.event.issue.labels.*.name, 'bug') }}
```

Comment:
Couldn't figure out how to negate so this is the only thing that I could think of